content/intro-to-storybook/angular/en/simple-component.md

```diff
@@ -161,7 +159,11 @@ module.exports = {
 -   '../src/**/*.stories.@(js|jsx|ts|tsx)'
 - ],
 + stories: ['../src/app/components/**/*.stories.ts'],
```

Comment:
I'm a bit confused here. The previous stories glob would have included the stories found by the new glob, and the prior prose on this page doesn't refer to addon-interactions... so why is the config change necessary?

Reply:
The addon interactions is going to be provided in the template hence why it's referenced. The glob is to prevent Storybook from picking up the default ones